Exiled.API/Features/DamageHandlers/AttackerDamageHandler.cs

```diff
@@ -115,5 +118,15 @@ public override void ProcessDamage(Player player)
                 Damage = 0f;
             }
         }
+
+        private string GetDebuggerDisplay()
+        {
+            var debugView = $"DamageHandler Damage = {Damage}";
```

Comment:
No `var` allowed.
Same goes for all occurrences.

Reply:
I just want to warn you that I found this line [Consuming.cs](https://github.com/Exiled-Team/EXILED/blob/84e496ffb9cc7c45c27b10c86a6989c88faad7ea/Exiled.Events/Patches/Events/Scp0492/Consuming.cs#L68) which is not mine but which contains a "var" when I was replacing the "var"s